### ISSUE TITLE
Keyboard Shortcuts Can't Be Set

### ISSUE THREAD

**Issue by dstark** [ankit/stylebot#696]
In the Chrome Web Store version of the extension, I seem unable to configure custom keyboard shortcuts from the "Basics" tab. I can type into the shortcut fields (e.g., "alt+s"). But this doesn't enable the shortcut. And I can't place my cursor in the field for the action I want to create the shortcut for, hold down the corresponding shortcut keys (e.g., "Alt" and "s"), and have that populate the shortcut field either.

(I'm using Chrome-based Edge and have tried the Edge extension. But when I come to sync styles with Google Drive, Google says the extension hasn't been verified and won't be authenticated with their OATH protocol.)

Would you have any suggestions about what I'm doing wrong or what might be amiss here?

Thank you so much!

**Comment by ankit** [comment]
@dstark i am not able to reproduce this. You are supposed to be able to enter `alt+s` and it should work

**Comment by dstark** [comment]
Thanks so much for your response!

I'm not able to set any keyboard shortcuts in the Stylebot Options page. If a screenshot helps, here's [one](http://www.jdavidstark.com/wp-content/uploads/file-store/Zotero%20Forums/Screenshot%202022-07-25%20093208.png) that shows how I don't get the customize option shown in [the documentation](https://stylebot.dev/help/#shortcuts).

I can try whatever key combinations I want in the various boxes, but they don't actually come up as saved. And I can type whatever I want to in them also, but (not surprisingly) that doesn't set the key combinations either.

Thanks, again!

**Comment by Waldemar7** [comment]
I know this is an old post, but I wanted to comment for people who have this issue. I had this at first (Firefox 137.0.1 64-bit, Windows 10). What fixed it for me was, disable and enable the extension after changing the shortcut.

It seems to me, the options dialog stores the changed setting somewhere, but doesn't apply it immediately.

**Comment by dstark** [comment]
> I know this is an old post, but I wanted to comment for people who have this issue. I had this at first (Firefox 137.0.1 64-bit, Windows 10). What fixed it for me was, disable and enable the extension after changing the shortcut.
> 
> It seems to me, the options dialog stores the changed setting somewhere, but doesn't apply it immediately.

Yep. Sure enough, @Waldemar7. The new shortcuts work for me also after disabling and reenabling the extension. Thanks so much for the tip!
